wandb/sdk/internal/internal_api.py

```diff
@@ -3504,6 +3504,39 @@ def create_artifact_type(
         _id: Optional[str] = response["createArtifactType"]["artifactType"]["id"]
         return _id
 
+    def change_collection_type(
```

Comment:
no need to use the internal api since we're keeping things in the user process. you can just add the mutation to the change function like this: https://github.com/wandb/wandb/blob/ef00e4f6b3be64a95a322353a874b818b9c901fa/wandb/apis/public/artifacts.py#L433-L466